lib/src/radio-group/radio-group.component.ts

```diff
@@ -0,0 +1,139 @@
+import { Component, Input, forwardRef, OnChanges, SimpleChanges, Provider } from "@angular/core";
+import { NG_VALUE_ACCESSOR, ControlValueAccessor } from "@angular/forms";
+
+export interface RadioGroupItem {
+    /** The label or text to be displayed in the list */
+    label: string;
+    /** optional description to be displayed next to the label */
+    description?: string;
+    /** any value which should be tied to the item */
+    value: any;
+    /** optional disabled flag. Will show the item grayed out and disabled */
+    disabled?: boolean;
+    /** The id or the unique key of the item */
+    key: string;
+}
+
+interface UniqueItem {
+    id: string;
+    optionItem: RadioGroupItem;
+    selected: boolean;
+}
+
+interface DisplayItem extends UniqueItem {
+    className: string;
+}
+
+const CUSTOM_RADIOGROUP_CONTROL_VALUE_ACCESSOR: Provider = {
+    provide: NG_VALUE_ACCESSOR,
+    useExisting: forwardRef(() => RadioGroupComponent),
+    multi: true,
+};
+
+@Component({
+    selector: "sebng-radio-group",
+    templateUrl: "./radio-group.component.html",
+    styleUrls: ["./radio-group.component.scss"],
+    providers: [CUSTOM_RADIOGROUP_CONTROL_VALUE_ACCESSOR],
+})
+export class RadioGroupComponent implements ControlValueAccessor, OnChanges {
+    @Input() list: Array<RadioGroupItem>; // TODO: Add support for custom html as well as string labels?
+    @Input() id?: string;
+    @Input() label?: string;
+    @Input() className?: string;
+    @Input() disabled?: boolean = false;
+    @Input() inline?: boolean = false;
```

Comment:
The list of inputs here doesn't match the ones in [react-components](https://github.com/sebgroup/react-components/blob/develop/src/RadioGroup/RadioGroup.tsx#LC12). Please make sure we have the same amount of features. 

Please note, that we moved away from using `id` as the name of the input. Additionally, you need to add an implementation of random IDs. Please install `@sebgroup/frontend-tools` to access the `randomId` method.

Reply:
As discussed on slack we will not be using randomIds yet